subsys/dfu/Kconfig

```diff
@@ -11,6 +11,9 @@ menuconfig DFU_TARGET
 
 if (DFU_TARGET)
 
+config DFU_TARGET_ALWAYS_RESTART
+	bool "Always start writing from the start"
```

Comment:
I'd like a longer explanation here about the implications of this. e.g. download restarted, pages erased, reboot?

Also, when is "always"?

Reply:
I do agree I just couldn't find the right phrasing. So what this option does is ensures that you'll always start downloading from the first byte. So offset 0.

This ensures that `fota_download` won't start downloading from a offset it got from `dfu_target_get_offset` on how far the previous download got and restart from there.